### ISSUE TITLE
Transaction history QoL (Order)

### ISSUE THREAD

**Issue by jiecut42** [decred/Paymetheus#314]
So this is just a quality of life issue with Paymetheus. Regarding the transaction history page. Whenever you want to check your newest transactions you need to go to the transaction history page and then you have to scroll all the way to the bottom.

Some solutions, if you think this is an issue, is to invert it so that newest transactions go at the top. You could also start the scroll bar somewhere else from the top. Or there could be a toggle on the date column to choose whether to sort by newest or oldest.

**Comment by scarmani** [comment]
Agreed, I have noticed the same, and wished for the ability to see newer transactions first by default, with transactions sortable by date.  Have also wished for the ability to export transactions as CSV.
